docs/docs/conceptual/choosing-a-cms.md

```diff
@@ -0,0 +1,69 @@
+
+Typically there are two reasons to intentionally take this approach:
+
+### Using Shopify as an e-commerce system
+
+Shopify has best-in-class e-commerce functionality, but the rest of their interface (for example, their blogs feature) is often seen as subpar compared to other systems. As a result, it's very standard to use Shopify as the backend for the store, but a general purpose CMS like Contentful as the backend for the rest of the site.
+
+### Using Wordpress as a blogging tool
+
+Because Wordpress is so familiar to content authors, and has a best-in-class content composition experience, it's quite common to move the blog portion of the website with Gatsby, and use a flexible content modelling CMS for the rest of the website.
+
+Apollo does this [for their blog](https://twitter.com/apollographql/status/1250479066605662210), as we do for the blog at [gatsbyjs.com/blog](gatsbyjs.com/blog).
+
+### Considerations when using multiple CMSs
+
+One of the key considerations when using content in multiple systems is that at some point one  content often needs to "know about" another system. For example, a landing page with content in Contentful may need to embed information about a specific product SKU (which lives in Shopify).
+
+The easiest way to create relationship references across CMSs is through one CMS storing unique IDs of content living in another CMS. In this case, you'd store an array of Shopify product IDs as a field of the relevant model in Contentful, then pull in the correct data via the appropriate queries in gatsby-node.js.
+
+## Other non-CMS options
+
+Finally, there are several options for content composition and management that work well as non-CMS choices.
+
+- **Markdown** and **MDX** are common choices for documentation and small developer sites, since they are natural composition formats for developers and allow embedding components within your content. There are several guides in our [Routing and Pages section](https://www.gatsbyjs.com/docs/how-to/routing/) on using Markdown and MDX. You may also want to consider a git-based CMS like NetlifyCMS or Foresty.io to provide a UI for this workflow.
+
+- **JSON** or **YAML** is a common choice for hierarchical data, for example, a site navigation tree, especially when the underlying content is stored in markdown.
+
+- **Web-based spreadsheets**, like **Airtable** or **Google Sheets**, are common use cases for tabular data. Airtable used this to [power their store locator](https://www.gatsbyjs.com/blog/2020-05-07-gatsby-delivers-impossible-burgers-map/), and [ProPublica used this as a database](https://www.gatsbyjs.com/blog/2019-03-29-interview-with-david-eads/) for interactive data journalism graphics.
```

Comment:
Can you update the link text to be more descriptive? So that if a screen reader users ended up on a link without the context of the rest of the sentence, they'd have an idea of where the link would take them.

Additional context: https://www.washington.edu/accessibility/links/

Reply:
Ah, thanks. Updated link text!